src/quo/components/wallet/swap_input/view.cljs

```diff
@@ -42,26 +44,65 @@
       [:container-style {:optional true} [:maybe :map]]]]]
    :any])
 
+(def icon-size 32)
+(def container-padding 24)
+(def max-cursor-position 5)
```

Comment:
How are we calculating the `container-padding` value? Is it possible to reference this from a style file?

Reply:
for `max-cursor-position`, is it possible that value will need to smaller on devices with smaller screens?

Reply:
> for `max-cursor-position`, is it possible that value will need to smaller on devices with smaller screens?

It is highly unlikely as 5 is a very low value, also this hanlde a very uncommon edge case 

> How are we calculating the `container-padding` value? Is it possible to reference this from a style file?

It is the sum of left and right padding. Updated the code to extract the values from the styles!